include/identity.php

```diff
@@ -452,33 +480,36 @@ function profile_sidebar($profile, $block = 0) {
 }
 
 
-function get_birthdays() {
-
+function get_birthdays()
+{
 	$a = get_app();
 	$o = '';
 
-	if (! local_user() || $a->is_mobile || $a->is_tablet)
+	if (! local_user() || $a->is_mobile || $a->is_tablet) {
 		return $o;
+	}
 
-//		$mobile_detect = new Mobile_Detect();
-//		$is_mobile = $mobile_detect->isMobile() || $mobile_detect->isTablet();
-
-//		if ($is_mobile)
-//			return $o;
+	/**
```

Comment:
you need have to look that comments don't start with `/**` because all stuff with `/**` will be added to the doxygen documentation.
Alternatives: `/*` or `/* *`

Reply:
Good catch.